packages/sitecore-jss-nextjs-editing-host/src/html-processors.ts

```diff
@@ -0,0 +1,30 @@
+
+import absolutify from './absolutify';
+
+export interface HtmlProcessor {
+  processHtml(html: string): string;
+}
+
+/**
+ * Absolutify html processor
+ * @constructor
+ * @param {string} publicUrl The public URL. This will be used when replacing relative links with absolute ones.
+ * @param {string[]} ignoredPaths URL path prefixes that should be ignored during link replacement.
+ */
+export class AbsolutifyHtmlProcessor implements HtmlProcessor {
+  public constructor(readonly publicUrl: string, readonly ignoredPaths?: string[]) {}
```

Comment:
@ambrauer It's not required to use `public` keyword, it will be `public` by default

Reply:
@sc-illiakovalenko Removed